Editor/src/com/kotcrab/vis/editor/plugin/PluginDescriptor.java

```diff
@@ -81,12 +81,19 @@ public PluginDescriptor (FileHandle file, Manifest manifest) throws EditorExcept
 		}
 
 		if (licenseFile != null) {
-			try {
-				JarFile jar = new JarFile(file.file());
+			JarFile jar = null;
+			try {				
+				jar = new JarFile(file.file());
 				JarEntry licenseEntry = jar.getJarEntry(licenseFile);
 				license = StreamUtils.copyStreamToString(jar.getInputStream(licenseEntry));
 			} catch (IOException e) {
 				throw new EditorException("Failed to read license file for plugin: " + file.extension(), e);
+			} finally {
+				try {
+					jar.close();
```

Comment:
This can be replaced by `IOUtils.closeQuietly(jar);` to avoid catching another try-catch with `IOException`. It isn't really important if it fails to close it.